rho/utilities.py

```diff
@@ -11,20 +11,17 @@
 """ A set of reusable methods common to many of the commands """
 
 from __future__ import print_function
+import csv
 import os
-import sys
 import re
-import csv
+import sys
 import tempfile
+import threading
```

Comment:
I've sorted the imports, if you think it is unnecessary noise for this PR I can revert and just add this line which is the relevant one.